iina.xcodeproj/project.pbxproj

```diff
@@ -523,6 +523,39 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		0507C25E2B5617650043AD05 /* he */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = he; path = he.lproj/MainMenu.strings; sourceTree = "<group>"; };
```

Comment:
The `KeyBinding.strings` file has been translated, but is missing from Xcode. As a result the names shown for keys in the `Key Bindings` settings tab are in English.

Reply:
This should be a simple fix; I can manually enable this file after merging this PR.

Reply:
Done